### PR TITLE
(models): Added missing gemini experimental models + fixed pricing for gemini-1.5-pro-exp-0827

### PR DESCRIPTION
**Pull request by F1bos**
This PR adds the missing experimental Gemini models [listed here](https://ai.google.dev/gemini-api/docs/models/experimental-models):

- gemini-flash-1.5-exp
- gemini-flash-8b-1.5-exp
- gemini-pro-1.5-exp

These models are currently free to use, likely due to heavy rate limiting and the lack of listed pricing [in the Google Cloud SKU list](https://cloud.google.com/skus/?&filter=gemini%20flash&currency=USD). As stated in the [Gemini API documentation](https://ai.google.dev/gemini-api/docs/models/gemini#experimental-models), experimental models are not intended for production use and can be changed without notice.

Additionally, the pricing for `gemini-1.5-pro-exp-0827` has been corrected to reflect its free status.

Pricing information and details on these models can be found on OpenRouter:

- [gemini-flash-1.5-exp](https://openrouter.ai/models/google/gemini-flash-1.5-exp)
- [gemini-flash-8b-1.5-exp](https://openrouter.ai/models/google/gemini-flash-8b-1.5-exp/providers)
- [gemini-pro-1.5-exp](https://openrouter.ai/models/google/gemini-pro-1.5-exp)

Note: Git is incorrectly showing changes to `gemini/gemini-pro` pricing. The actual changes are for `gemini/gemini-1.5-pro-exp-0801`. This seems to be a Git diff issue.

### CODE REVIEW

File: litellm/model_prices_and_context_window_backup.json [hunk 1]
```diff
@@ -2674,10 +2716,10 @@
         "max_tokens": 8192,
         "max_input_tokens": 2097152,
         "max_output_tokens": 8192,
-        "input_cost_per_token": 0.0000035,
-        "input_cost_per_token_above_128k_tokens": 0.000007,
-        "output_cost_per_token": 0.0000105,
-        "output_cost_per_token_above_128k_tokens": 0.000021,
+        "input_cost_per_token": 0,
```
Comment (ghost) [inline litellm/model_prices_and_context_window_backup.json:2719]: is gemini-pro free?

Comment (ghost) [reply litellm/model_prices_and_context_window_backup.json:2719]: isn't this the pricing? https://ai.google.dev/pricing
Screenshot 2024-09-13 at 9 12 19 PM

Comment (F1bos) [reply litellm/model_prices_and_context_window_backup.json:2719]: I did not change the pricing for Gemini Pro. That's a Git bug. If you expand the changes, you will see that I changed "gemini/gemini-1.5-pro-exp-0827".

**Comment by ghost** [comment]
LGTM. Thanks for the PR @F1bos, do you have a linkedin/twitter? Happy to give a shoutout in the daily changelog.

**Comment by F1bos** [comment]
> LGTM. Thanks for the PR @F1bos, do you have a linkedin/twitter? Happy to give a shoutout in the daily changelog.

Nope, you can just mention my github profile. Thanks.

**Pull request merged**